helix-view/src/document.rs

```diff
@@ -141,6 +141,8 @@ pub struct Document {
     pub inlay_hints_oudated: bool,
 
     path: Option<PathBuf>,
+    /// Path without normalization or canonicalization
+    raw_path: Option<PathBuf>,
```

Comment:
I could have done without changing the existing signatures, but it looked terrible.